concepticondata/conceptlists/Gabelentz-1892-1074.tsv

```diff
@@ -0,0 +1,1075 @@
+Gabelentz-1892-1074-886	886	Gähnen	to yawn	VIII. Verba	C. Leben, Körperfunctionen	1028	YAWN
+Gabelentz-1892-1074-887	887	Schlafen	to sleep	VIII. Verba	C. Leben, Körperfunctionen	1585	SLEEP
+Gabelentz-1892-1074-888	888	Wach sein	to be awake	VIII. Verba	C. Leben, Körperfunctionen	825	AWAKE
+Gabelentz-1892-1074-889	889	Erwachen	to wake up	VIII. Verba	C. Leben, Körperfunctionen	1602	WAKE UP
+Gabelentz-1892-1074-890	890	Ruhen	to rest	VIII. Verba	C. Leben, Körperfunctionen	168	REST
+Gabelentz-1892-1074-891	891	Leiden, dulden	to suffer	VIII. Verba	C. Leben, Körperfunctionen	1101	SUFFER
+Gabelentz-1892-1074-892	892	Schmerzen, Schmerz empfinden	to feel pain	VIII. Verba	C. Leben, Körperfunctionen	2561	HURT (SENSE PAIN)
+Gabelentz-1892-1074-893	893	Heilen	to heal	VIII. Verba	C. Leben, Körperfunctionen	499	CURE
+Gabelentz-1892-1074-894	894	Weinen	to cry	VIII. Verba	C. Leben, Körperfunctionen	1839	CRY
+Gabelentz-1892-1074-895	895	Lachen	to laugh	VIII. Verba	C. Leben, Körperfunctionen	1355	LAUGH
+Gabelentz-1892-1074-896	896	Küssen	to kiss	VIII. Verba	C. Leben, Körperfunctionen	578	KISS
+Gabelentz-1892-1074-897	897	Lecken	to lick	VIII. Verba	C. Leben, Körperfunctionen	319	LICK
+Gabelentz-1892-1074-898	898	Schwitzen	to sweat	VIII. Verba	C. Leben, Körperfunctionen	126	SWEAT (PERSPIRE)
+Gabelentz-1892-1074-899	899	Tödten	to kill	VIII. Verba	C. Leben, Körperfunctionen	1417	KILL
+Gabelentz-1892-1074-900	900	Sehen	to see	VIII. Verba	C. Leben, Körperfunctionen	1409	SEE
+Gabelentz-1892-1074-901	901	Hören	to hear	VIII. Verba	C. Leben, Körperfunctionen	1408	HEAR
+Gabelentz-1892-1074-902	902	Kosten	to taste	VIII. Verba	C. Leben, Körperfunctionen	21	TASTE (SOMETHING)
+Gabelentz-1892-1074-903	903	Riechen, schnüffeln	to smell	VIII. Verba	C. Leben, Körperfunctionen	1586	SMELL (PERCEIVE)
+Gabelentz-1892-1074-904	904	Fühlen	to feel	VIII. Verba	C. Leben, Körperfunctionen	1079	FEEL
+Gabelentz-1892-1074-905	905	Berühren, betasten	to touch	VIII. Verba	C. Leben, Körperfunctionen	1892	TOUCH
+Gabelentz-1892-1074-906	906	Den Beischlaf vollziehen	to have intercourse	VIII. Verba	C. Leben, Körperfunctionen	1785	HAVE SEX
+Gabelentz-1892-1074-907	907	Gehen	to go	VIII. Verba	D. Gehen, Kommen u.s.w.	695	GO
+Gabelentz-1892-1074-908	908	Kommen	to come	VIII. Verba	D. Gehen, Kommen u.s.w.	1446	COME
+Gabelentz-1892-1074-909	909	Besuchen	to visit	VIII. Verba	D. Gehen, Kommen u.s.w.	1377	VISIT
+Gabelentz-1892-1074-910	910	Begegnen	to meet	VIII. Verba	D. Gehen, Kommen u.s.w.	1831	MEET
+Gabelentz-1892-1074-911	911	Landen	to land	VIII. Verba	D. Gehen, Kommen u.s.w.	1922	LAND (DESCEND)
+Gabelentz-1892-1074-912	912	Absegeln	to start sailing	VIII. Verba	D. Gehen, Kommen u.s.w.		
+Gabelentz-1892-1074-913	913	Folgen	to follow	VIII. Verba	D. Gehen, Kommen u.s.w.	1085	FOLLOW
+Gabelentz-1892-1074-914	914	Verfolgen	to pursue	VIII. Verba	D. Gehen, Kommen u.s.w.	1047	PURSUE
+Gabelentz-1892-1074-915	915	Erreichen	to arrive at a destination	VIII. Verba	D. Gehen, Kommen u.s.w.	1387	ARRIVE
```

Comment:
That rather means *REACH, usually.

Reply:
Might be that I'm overlooking something obvious right now, but what would the conceptual difference be?

Reply:
Ah sorry, I meant reach in the sense of succeeding, reaching a goal (including the goal of reaching a place) where as ARRIVE is much more neutral.

Reply:
I see, but it doesn't feel to me like there's a difference in meaning or emphasis between e.g. "die Stadt erreichen" und "in der Stadt ankommen". Leaving this open for now.

Reply:
There are instances of _reach_ AND _arrive_ mapped to the concept ARRIVE. I'd leave it as is.